assets/js/events-category.js

```diff
@@ -13,6 +13,7 @@ jQuery( document ).ready(function( $ ) {
 		}
 	}
 
-	$(".nhsuk-width-container").on( "change", "#select-1", onCatChange );
+	//$(".nhsuk-width-container").on( "change", "#select-1", onCatChange );
+  //this won't work anymore - we don't use nhsuk classes anymore
```

Comment:
Has the original impact of onChange been made redundant or, do we still need to listen for an onChange event?

Reply:
I wasn't making JS changes, so commented this out because it used a dead class - I don't know what it was for so wasn't sure enough to delete.  I would strongly suspect, however, that is isn't needed anymore.